docs/source/conf.py

```diff
@@ -48,6 +48,12 @@
     'recommonmark',
 ]
 
+# Hack to make sphinx_js work with python 3.10
+import collections
+import collections.abc
+if not hasattr(collections, 'Mapping'):
+    collections.Mapping = collections.abc.Mapping
```

Comment:
My approach was to [fork sphinx-js](https://github.com/Dallinger/sphinx-js). The monkey-patch is definitely a lot less work, and I don't feel as uncomfortable with it as I thought I would, but I'm still undecided. Thoughts?

Reply:
If this change affected the dallinger codebase I'd be slightly more concerned, but it's only effective in the docs generation, so I believe it's safe.